extensions/container-image/deployment/src/main/java/io/quarkus/container/image/deployment/devui/ContainerImageDevUiProcessor.java

```diff
@@ -37,7 +37,7 @@ CardPageBuildItem create(List<AvailableContainerImageExtensionBuildItem> extensi
         return card;
     }
 
-    @BuildStep(onlyIf = IsDevelopment.class)
+    @BuildStep
```

Comment:
Unlike other extensions, this method not only produces the build item, but also calls `DevConsoleManager.register()`. I _guess_ it's _probably_ OK, but I'm not exactly sure.